cmd/setup.go

```diff
@@ -834,6 +834,7 @@ func configureTariffs(conf globalconfig.Tariffs) (*tariff.Tariffs, error) {
 }
 
 func configureDevices(conf globalconfig.All) error {
+	// TODO: add name/identifier to error for better highlighting in UI
```

Comment:
@naltatis wird doch mal einen Blick in `configureMeters`. Das TODO kann weg...

Reply:
Das war übrigens die Diskussion zum RCT vom Freitag wo wir genau das ausprobiert hatten ;)

Reply:
Ok, den `device` Key hatte ich übersehen. Das funktioniert, der kommt auch an und wird jetzt im UI berücksichtigt. Allerding bekomme ich im Fehlerfall keine DB-Devices und der `/api/config/site` Endpunkt funktioniert nicht was dazu führt, dass ich keine zugeordneten Geräte anzeigen kann.

Reply:
Ah, sehe gerade, das wir dafür schon Todos geschrieben und auf "Out of scope" gestellt haben. Dann ignorier das einfach. Die Fehlerkommunikation funktioniert jetzt auf jeden Fall (wenn dann die Daten kommen würden).